dotnet/Capstone/Controllers/BirdController.cs

```diff
@@ -71,35 +73,117 @@ public Bird getRandomBird()
             return randomBird;
         }
 
-        [HttpPost("/birds")]
-        public void createBird(Bird newBird)
+        [HttpPost("/lists/{listId}/addBird")]
```

Comment:
Nothing currently stopping a user from adding a bird to any other user's list. Not a big deal for the front end, but always good to secure the back end. I don't mind if that update ends up being a different PR.

If you do update now, I recommend using the 403 code (`Forbid()` method). I've mistakenly used the 401 unauthorized error code for similar situations, but 403 is meant to indicate that the user is logged in, just not allowed to do that action whereas 401 is meant to indicate the user recognized.

Reply:
This has been solved since the list displayed is only from the logged in user.